tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -293,9 +292,8 @@ def test_45523(self):
         LOGGER.info("STARTED: Test Delete source object when copy multipart object is in progress")
         LOGGER.info("Upload a large object (~1GB) srcobj to srcbuck")
         self.create_put_parallel_copy_and_delete_object((self.srcbuck, "src-obj", self.destbuck,
-                                                         "dest-obj", errmsg.NO_SUCH_KEY_ERR, 4,
-                                                         self.file_path),
-                                                        (self.srcbuck, "src-obj"),
+                                                        "dest-obj", errmsg.NO_SUCH_KEY_ERR, 4,
```

Comment:
Let's fix the indentation of the continuation lines here. Moving the next tuple arg to a newline will improve readability as well.
```suggestion
                                                         "dest-obj", errmsg.NO_SUCH_KEY_ERR, 4,
                                                         self.file_path),
                                                        (self.srcbuck, "src-obj"),
```